registry/consul/registry.go

```diff
@@ -187,5 +187,10 @@ func (r *consulRegistry) IsAvailable() bool {
 
 // Destroy consul registry center
 func (r *consulRegistry) Destroy() {
+	if r.URL != nil{
```

Comment:
You should change test case for ensure this code become effective

Reply:
OK